functions.php

```diff
@@ -17,7 +17,7 @@ function createDBConnection(): PDO {
  * @return array
  */
     function fetchAllRecipes($db): array {
-        $query = $db->prepare('SELECT `recipe`, `cuisine`, `time`, `link` FROM `recipes`;');
+        $query = $db->prepare("SELECT `recipe`, `cuisine`, `time`, `link` FROM `recipes` WHERE `deleted` = '0';");
```

Comment:
if you want to use single quotes on the query you can just put the 0 in without quotes. Not sure which is best practice though.

Reply:
I tend to go with single quotes on queries as a standard just in case I add any string values - sort of feels consistent?